backend/app/stream.py

```diff
@@ -26,11 +28,20 @@ async def astream_messages(
             yield root_run_id
         elif event["event"] == "on_chain_stream" and event["run_id"] == root_run_id:
             new_messages: list[BaseMessage] = []
-            for msg in event["data"]["chunk"]:
-                if msg.id in messages and msg == messages[msg.id]:
+
+            # event["data"]["chunk"] is a Sequence[AnyMessage] or a Dict[str, Any]
+            state_chunk_msgs: Union[Sequence[AnyMessage], Dict[str, Any]] = event[
+                "data"
+            ]["chunk"]
+            if isinstance(state_chunk_msgs, Dict):
```

Comment:
this should be lowercase dict

Reply:
Good catch. I'll update all other instances of this error.